soc/intel/intel_adsp/ace/Kconfig.defconfig.series

```diff
@@ -60,6 +60,20 @@ config LOG_BACKEND_ADSP
 
 endif # LOG
 
+config XTENSA_MORE_SPIN_RELAX_NOPS
+	default y if SMP && MP_MAX_NUM_CPUS > 1
+
+if XTENSA_MORE_SPIN_RELAX_NOPS
+
+config XTENSA_NUM_SPIN_RELAX_NOPS
+	default 32 if MP_MAX_NUM_CPUS = 1
+	default 64 if MP_MAX_NUM_CPUS = 2
+	default 96 if MP_MAX_NUM_CPUS = 3
+	default 128 if MP_MAX_NUM_CPUS = 4
+	default 160 if MP_MAX_NUM_CPUS = 5
```

Comment:
Pedantry, but ACE doesn't exist with most of these CPU counts.  Seems like needless abstraction?  Why not just one line per SOC specifying the right number of NOPs for that platform.

Further pedantry: MP_MAX_NUM_CPUS==1 is a uniprocessor configuration (even though technically being built as an SMP kernel, which is legal but weird) and there's no one else on the bus to contend with, should be zero in that case, not 32.

Reply:
I wish that kconfig had a way to do `(32 * num_cpus)`. Though I just want this to be generalized here using `CONFIG_MP_MAX_NUM_CPUS`. So until the audio chips has more than 5, we are covered for any future iterations of SoCs.

Reply:
It would require changing the pre-existing SPIN_RELAX_NOPS API, but you could do it as a "nops per cpu" value and then expand the multiplication as part of the C preprocessor code emitting the assembly.  Probably more work than necessary here, but might scale better and be a more obvious fit for other Xtensa platforms.